### DISCUSSION TITLE
How to ignore logs of terminal, using "session_log" in netmiko? [eltex]

### DISCUSSION THREAD

**Discussion by leader01** [ktbyers/netmiko#2644] [Q&A]
There is one switch with faulty fastethernet port and terminal constantly gives logs that prevent my parsing.
There is my question on Stackoverflow: https://stackoverflow.com/questions/71044122/how-to-ignore-annoying-logs-of-terminal-using-session-log-in-netmiko

**Comment by ktbyers** [comment]
@leader01 Can you tell the switch to stop logging messages to the terminal. 

That would be the standard fix for this type of thing i.e. the switch should not be logging these messages to the terminal (should only be logging them to the logging buffer or to an external logging server).
